src/logging/json.rs

```diff
@@ -279,7 +282,18 @@ where
             }
 
             for (key, val) in &self.static_fields {
-                ser.serialize_entry(key, val)?;
+                if !self.parse_env_in_static {
```

Comment:
Would it be better (and faster) to resolve environment variables only once when creating the formatter?